webpages/settings/index.js

```diff
@@ -248,6 +248,41 @@ chrome.runtime.sendMessage("getSettingsInfo", ({ manifests, addonsEnabled, addon
     manifest._tags.forEditor = manifest.tags.includes("theme") && manifest.tags.includes("editor");
     manifest._tags.forWebsite = manifest.tags.includes("theme") && manifest.tags.includes("community");
   }
+  setTimeout(function () {
+    let keyInputs = document.querySelectorAll(".key");
+    for (let i = 0; i < keyInputs.length; i++) {
+      keyInputs[i].addEventListener("keydown", function (e) {
+        e.preventDefault();
+        e.target.value = e.ctrlKey
+          ? "Ctrl" +
+            (e.shiftKey ? " + Shift" : "") +
+            (e.key == "Control" || e.key == "Shift"
+              ? ""
+              : (e.ctrlKey ? " + " : "") +
+                (e.key.toUpperCase() === e.key
+                  ? e.code.includes("Digit")
+                    ? e.code.substring(5, e.code.length)
+                    : e.key
+                  : e.key.toUpperCase()))
+          : "";
+        for (let j = 0; j < vue.manifests.length; j++) {
+          if (vue.manifests[j].settings) {
+            for (let k = 0; k < vue.manifests[j].settings.length; k++) {
+              if (vue.manifests[j].settings[k].id == e.target.id) {
+                vue.updateOption(e.target.id, e.target.value, vue.manifests[j]);
+                objectVals = Object.values(vue.addonSettings[vue.manifests[j]._addonId]);
+                return;
+              }
+            }
+          }
+        }
+      });
+      keyInputs[i].addEventListener("keyup", function (e) {
+        e.target.value =
+          e.target.value == "Ctrl" || objectVals.filter((v) => v === e.target.value).length > 1 ? "" : e.target.value;
+      });
```

Comment:
What does this event do?

Reply:
e.target.value == "Ctrl" || objectVals.filter((v) => v === e.target.value).length > 1 ? "" : e.target.value;
user only pressed control key OR another setting uses that hotkey then make input empty. Otherwise keep as is.

Reply:
Please let's keep talking on the Discord server, I have no idea what `objectVals` means on here.